src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -108,10 +108,10 @@ const useGroupInfo = () => {
     launchGallery();
   };
 
-  const uploadImageBase64 = async (res) => {
+  const uploadImage = async (pathImg) => {
     try {
       setIsUploadingImage(true);
-      const result = await uploadFile(`data:image/jpeg;base64,${res.base64}`);
+      const result = await ImageUtils.uploadImage(pathImg);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `uploadImage` now takes a path to an image file as an argument instead of a base64 string. Make sure that the `uploadImage` function in `ImageUtils` handles this change appropriately, and that all calls to this function have been updated to pass the correct argument type.



<!-- This is an auto-generated comment by OSS CodeRabbit -->